### DISCUSSION TITLE
How can we use VISP algorithms to combine both: tracking and IBVS to do a pick-and place with a robotic arm ?

### DISCUSSION THREAD

**Discussion by AssanhajiMohamed** [lagadic/visp#1171] [Q&A] [answered]

**Comment by s-trinh** [comment, marked as answer]
For IBVS:
- https://visp-doc.inria.fr/doxygen/visp-daily/tutorial-ibvs.html
- [and these tutorials for some examples with different robots](https://visp-doc.inria.fr/doxygen/visp-daily/tutorial_vs.html)

---

If PBVS is also suitable/preferable, you can start with these tutorials:
- for [model-based tracking](https://visp-doc.inria.fr/doxygen/visp-daily/tutorial-tracking-mb-generic.html)
- for [model-based tracking with an RGB-D camera](https://visp-doc.inria.fr/doxygen/visp-daily/tutorial-tracking-mb-generic-rgbd.html)
- for [tracking with an AprilTag](https://visp-doc.inria.fr/doxygen/visp-daily/tutorial-detection-apriltag.html)

You can look at [these videos](https://www.youtube.com/@VispTeam/videos) for some demos.

---

Do not hesitate if you have a more specific question.
